config/recaptcha.yml

```diff
@@ -0,0 +1,5 @@
+development:
```

Comment:
These are keys to the development version of our Recaptcha. I will regenerate keys for production and will insert this yml file during deploy.